src/EnergyPlus/ZoneEquipmentManager.hh

```diff
@@ -122,9 +122,20 @@ namespace ZoneEquipmentManager {
                              Real64 &FinalTotalReturnMassFlow // Final total return air mass flow rate
     );
 
+    void CalcZoneInfiltrationFlows(EnergyPlusData &state,
+                                   int const ZoneNum,                // current zone index
+                                   Real64 &ZoneReturnAirMassFlowRate // zone total zone return air mass flow rate
+    );
+
+    void ZoneReturnFlowsMaximum(EnergyPlusData &state,
+        int const ZoneNum,
+        Real64 &MaximumZoneReturnMassFlow // maximum zone total return air mass flow rate
+    );
+
     void CalcAirFlowSimple(EnergyPlusData &state,
                            int const SysTimestepLoop = 0,              // System time step index
-                           bool const AdjustZoneMixingFlowFlag = false // flags to adjust zone mxing mass flow rate
+                           bool const AdjustZoneMixingFlowFlag = false, // flags to adjust zone mxing mass flow rate
+                           bool const AdjustZoneInfiltrationFlowFlag = false // flags to djust zone infiltration air flow rate
```

Comment:
Replaced the "AdjustZoneMixingFlowFlag" flag, which was used for zone mixing and infiltration adjustment with two different flags one for zone mixing and another for infiltration.